Spanner/tests/Unit/Connection/GrpcTest.php

```diff
@@ -124,11 +124,41 @@ public function testListInstances()
 
     public function testGetInstance()
     {
+        $fieldNames = [];
```

Comment:
After my other comments are addressed, I believe this particular test method can be reverted.

Reply:
Yes, reverted